ocs_ci/helpers/ocs_metrics_exporter_helpers.py

```
@@ -0,0 +1,1047 @@
+                    "command": ["/bin/sh", "-c", "sleep 3600"],
+                    "volumeMounts": [{"mountPath": "/data", "name": "pvc-vol"}],
+                }
+            ],
+            "volumes": [
+                {"name": "pvc-vol", "persistentVolumeClaim": {"claimName": pvc_name}}
+            ],
+        },
+    }
+    pod_obj = pod_helpers.create_pod(**pod_dict)
+    pod_helpers.wait_for_resource_state(pod_obj, constants.STATUS_RUNNING, timeout=300)
+    logger.info(f"Created pod {pod_name} with PVC {pvc_name} in namespace {namespace}")
```

Ruff (0.15.21)

[warning] Logging statement uses f-string

(G004)